packages/ActionBar/stories/ShowcaseApp/ShowcaseApp.js

```diff
@@ -69,16 +69,10 @@ export default function App() {
     data,
     maxSortFields: 3,
   });
-  const { orderedColumnIds, isColumnHidden, ...handlers } = useColumnsArrangement([
-    "goals",
-    "name",
-    "status",
-    "country",
-    "joined",
-    "shareable",
-    "level",
-    "position",
-  ]);
+  const { orderedColumnIds, isColumnHidden, ...handlers } = useColumnsArrangement(
+    ["goals", "name", "status", "country", "joined", "shareable", "level", "position"],
```

Comment:
have you considered to use an object instead of parameters for useColumnArragment?

const order = ["goals", "name", "status", "country", "joined", "shareable", "level", "position"];
const disable = ["name"]

```js
const { orderedColumnIds, isColumnHidden, ...handlers } = useColumnsArrangement({ order, disable })
```

I think would be more flexible and future resilience in case we want to add more parameters. 

but I'm not super hard on this just a suggestion.

Reply:
I like it! Maybe a starling day for grouping the handlers/props returned in another object(`const {filteredData, filterProps} = useFilter()`), so they can be released together as a breaking change